.bundle/config

```diff
@@ -1,4 +1,5 @@
 ---
 BUNDLE_PATH: "vendor/bundle"
-BUNDLE_JOBS: "16"
+BUNDLE_JOBS: "3"
+BUNDLE_RETRY: "3"
```

Comment:
Since this involves a configuration change we could omit the change from this PR.

Reply:
> Great work @mertGokduman 👍
> I tested your implementation and everything works as expected 🎉
> The code changes are also consistent and overall look good ⭐
> I've only left a couple of comments ([1](https://github.com/wordpress-mobile/WordPress-iOS/pull/17911#discussion_r802586354), [2](https://github.com/wordpress-mobile/WordPress-iOS/pull/17911#discussion_r802589388)) that along with @AliSoftware's [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/17911#discussion_r801884312) would be nice to handle before merging to the codebase.
> Thanks again for stepping forward to help with the `WordPress-iOS` implementation 🙇

I will remove the my translation part :)

Reply:
Hi @antonis ,
Actually, I don't know how to remove this part from PR. I guess if I will delete this part from my project in my computer, build will fail. Can you help me for this issue ?

Reply:
@mertGokduman You can just run `git restore .bundle/config` to revert the changes made to this file in your PR.

FYI, this change is a side effect of how `bundle` operates depending on the machine (number of CPUs of your Mac, etc) your run it from. This is likely why that change happened when you worked on this PR locally and ran `bundle exec` (or `rake dependencies` which uses `bundle exec` under the hood).

Reply:
@AliSoftware Thank you for your answer. 🙏

Reply:
@AliSoftware  git restore .bundle/config this one did not work for my case :) Is there any other solutions for this?

Reply:
Done :) I used git checkout origin/trunk -- .bundle/config .

Reply:
Ah right, of course, wrong command, my bad 🤦 (it would only have worked if you didn't commit the file yet)

So we need to tell git to pull the version of that file that is in the commit or branch before your changes. For example, ask git to restore it to the content of the file as it is in `trunk`, before your PR:

```
git checkout trunk .bundle/config 
```
Then commit and push.